### PR TITLE
Feed/2 late code

### PR DESCRIPTION
**Pull request by Sirgot**

### CODE REVIEW

File: bacterialand-server/src/js/bacterium-be/game/index.js [hunk 1]
```diff
@@ -69,14 +88,82 @@ async function createGame(context, req) {
...
+        let parser = ''.concat(turn, '16');
+        const collection = await client.db('bacterium-store').collection('game');
+        collection.findOneAndUpdate({
+                _id: ObjectId(req.body.gameId),
+            // How add the 5 6 7 an seven checks?
+                turn: req.body.turn,
+                turn: {
+                    $nin: req.body.move
+                },
+                parser: {$exists: false}
```
Comment (jarpi) [inline bacterialand-server/src/js/bacterium-be/game/index.js:143]: I believe it should be [parser]
where parser = 'id-gamer-guid.16'

File: bacterialand-server/src/js/bacterium-be/game/index.js [hunk 2]
```diff
@@ -69,14 +88,82 @@ async function createGame(context, req) {
...
+ *  8 updateamos
+ *  9 devolver estado
+ *
+ * @param context
+ * @param req
+ * @returns {Promise<void>}
+ */
+async function updateGame(context, req) {
+
+    if (makeMove(context, req)) {
```
Comment (jarpi) [inline bacterialand-server/src/js/bacterium-be/game/index.js:125]: No need to check move. Use findOneAndUpdate body query

File: bacterialand-server/src/js/bacterium-be/game/index.js [hunk 3]
```diff
@@ -69,14 +88,82 @@ async function createGame(context, req) {
...
+                parser: {$exists: false}
+            },
+            {
+                "$set": {
+
+                    turn: req.body.turn
+                }
+            },
+            {
+                returnDocument: true
```
Comment (jarpi) [inline bacterialand-server/src/js/bacterium-be/game/index.js:152]: returnNewDocument

**Comment by Sirgot** [comment]
un sync of the logic

**Pull request merged**